lib/hanami/http/status.rb

```diff
@@ -21,7 +21,13 @@ class Status
       #
       # @since 2.0.2
       # @api private
-      SYMBOLS = ::Rack::Utils::SYMBOL_TO_STATUS_CODE
+      SYMBOLS = ::Rack::Utils::SYMBOL_TO_STATUS_CODE.dup
+
+      if Gem.loaded_specs["rack"].version.version.start_with?("3")
+        SYMBOLS[:unprocessable_entity] = 422
+      else
+        SYMBOLS[:unprocessable_content] = 422
```

Comment:
How do we feel about keeping `:unprocessable_entity` in **addition** to the new symbol for backwards compatibility?

Reply:
I think that because we've [documented `unprocessable_entity`](https://guides.hanamirb.org/v2.1/actions/status-codes/), it would be good for us to maintain it in the future.

I think for our 2.3 docs, we should show _both_ `unprocessable_content` and `unprocessable_content` as working for 422, and should update this PR to work with both, for either version of Rack.

This feels important during this Rack v2/v3 transitional phase, which I expect we might keep for ~a year or so.

I _think_ this might also simplify this PR? We'd just merge unprocessable_content and unprocessable_content keys into our `SYMBOLS` hash regardless of the version of rack we see being used?